frontend/src/core/comments/components/TeamComment.js

```diff
@@ -0,0 +1,60 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './CommentList.css'
```

Comment:
That looks fishy. Why do you import that CSS file, instead of the module itself? Can you make it so you use the `CommentsList` module directly?

Reply:
(Also, it should be `Comment*s*List.css` ;) )

Reply:
`CommentsList` renders differently than `TeamComments` in regards to users who are not authorized to leave comments and what renders when there are no comments. However, they share most of the CSS for displaying the list of comments and the add comment textarea.

I can copy the styling into the `TeamComments.css` file but that doesn't feel right either. 
@mathjazz Do you have a suggestion here?

Reply:
I think you should be able to modify `CommentsList` such that you could use it in `TeamComments`.